docs/insiders/getting-started.md

```diff
@@ -30,20 +30,31 @@ your GitHub account in order to access the Insiders repository programmatically
   [Generate a new token]: https://github.com/settings/tokens/new
   [scopes]: https://docs.github.com/en/developers/apps/scopes-for-oauth-apps#available-scopes
 
+Some of the instructions below require that the `GH_TOKEN` environment
+variable is set to the value of the personal access token you
+generated in the previous step. Note that the personal access token
+must be kept secret at all times, as it allows the owner to access
+your private repositories.
+
 ## Installation
 
 ### with pip
 
-Material for MkDocs Insiders can be installed with `pip`:
```

Comment:
I think we should move this into content tabs, similar to what we did in our getting started guide:
https://squidfunk.github.io/mkdocs-material/getting-started/#with-pip-9x

Reply:
You mean the complete pip/Docker/git alternatives?

Reply:
No, the "Latest / 9.x" content tabs ☺️

<img width="643" alt="Bildschirm­foto 2023-09-27 um 20 36 31" src="https://github.com/squidfunk/mkdocs-material/assets/932156/484c6d8e-cf05-4d20-b0b1-70b09f816ad6">

Reply:
Funny, that is what I had at first then then thought that it was not necessary. Will re-create that version for consistency's sake. Updating in a mo

Reply:
I am also looking at the instructions for upgrading, adding instructions for git there as well. Got the text and will go about testing the instructions before committing. Is there anything else that should be done while I am at it?

Reply:
From the top of my head:

- Installing Insiders Latest
- Installing Insiders specific tag
- Upgrading Insiders

We should probably adjust the structure to mirror what we have here:
https://squidfunk.github.io/mkdocs-material/changelog/

- Changelog (section)
- How to upgrade

Reply:
Like so?

<img width="978" alt="image" src="https://github.com/squidfunk/mkdocs-material/assets/4134224/0afc20ed-3671-4e87-b52e-d65806a6d2d1">

Reply:
Ok, pushed those changes, hoping they match what you imagined. Tested the upgrade (and downgrade!) instructions and they should work. You may want to sanity check. 

On thing I note is that for the `git` version there does not seem to be a need to specify `--upgrade`, presumably because of the `-e`? I would leave it out in case it gets in the way of a downgrade. I am not that deeply familiar with how `pip` works in cases like this.